.github/workflows/enos-verify-stable.yml

```diff
@@ -6,14 +6,17 @@ on:
       - enos-verify-stable
       - enos-verify-stable::*
 
+env:
```

Comment:
The env context isn't allowed in jobs.with. We'll need to spawn another job to determine the matrix name and pass it through.